src/utils/customElements.ts

```diff
@@ -23,7 +23,10 @@ const AugmentableElementsSelector = `${ElementsWithPlusButtonSelector}, ${Elemen
  * @returns {NodeListOf<HTMLElement>}
  */
 export function getAugmentableDescendants(node: HTMLElement | Document): NodeListOf<HTMLElement> {
-  return node.querySelectorAll(AugmentableElementsSelector);
+  const augmentableElementsSelector =
+    `${ElementSelector}` + (isInsideIFrame() ? `, ${ElementsAugmentableOnlyInIframe}` : '');
```

Comment:
nitpick: I think a simple `if` condition would be more readable and clear than conditional string concatenation. What do you think?
```
if (isInsideIframe()) {
  return node.querySelectorAll(AugmentableElementsSelector);
}
return node.querySelectorAll(ElementSelector);
```

Reply:
That does truly look way better, thanks for the suggestion. 👍